pkg/kv/kvserver/kvstorage/destroy.go

```diff
@@ -32,3 +56,33 @@ type ClearRangeDataOptions struct {
 	// heuristic will be used instead.
 	MustUseClearRange bool
 }
+
+// ClearRangeData clears the data associated with a range descriptor selected
+// by the provided clearRangeDataOptions.
```

Comment:
`ClearRangeDataOptions`, or (as a general approach resilient to this kind of out-of-sync-ness) "by the provided options".